bitcoin/examples/sign-tx-segwit-v0.rs

```diff
@@ -66,10 +66,10 @@ fn main() {
 
     // Sign the sighash using the secp256k1 library (exported by rust-bitcoin).
     let msg = Message::from(sighash);
-    let sig = secp.sign_ecdsa(&msg, &sk);
+    let signature = secp.sign_ecdsa(&msg, &sk);
 
     // Update the witness stack.
-    let signature = bitcoin::ecdsa::Signature { sig, hash_ty: EcdsaSighashType::All };
+    let signature = bitcoin::ecdsa::Signature { signature, sighash_type };
```

Comment:
Good idea to just re-use the assignment statement to `sighash_type` from earlier.

Reply:
I guess you could also eliminate the assignment statement and use `EcdsaSighashType::All` everywhere instead also.

Reply:
I think it's helpful to have the separate variable. But I could go either way.